llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -796,7 +796,8 @@ class AMDGPUImageDimIntrinsicEval<AMDGPUDimProfile P_> {
 // All dimension-aware intrinsics are derived from this class.
 class AMDGPUImageDimIntrinsic<AMDGPUDimProfile P_,
                               list<IntrinsicProperty> props,
-                              list<SDNodeProperty> sdnodeprops> : DefaultAttrsIntrinsic<
+                              list<SDNodeProperty> sdnodeprops,
+                              string name = ""> : Intrinsic<
```

Comment:
Changing the attributes of the existing intrinsics should be done separately

Reply:
I'll create a PR for that.